src/main/host/syscall/format.rs

```diff
@@ -120,12 +169,15 @@ macro_rules! simple_array_impl {
             fn fmt(
                 &self,
                 f: &mut std::fmt::Formatter<'_>,
+                options: FmtOptions,
                 mem: &MemoryManager,
             ) -> std::fmt::Result {
-                match mem.memory_ref(TypedPluginPtr::new::<$type>(self.ptr, K)) {
-                    Ok(vals) => write!(f, "{:?} ({:p})", &(*vals), self.ptr),
+                match (options, mem.memory_ref(TypedPluginPtr::new::<$type>(self.ptr, K))) {
+                    (FmtOptions::Standard, Ok(vals)) => write!(f, "{:?} ({:p})", &(*vals), self.ptr),
+                    (FmtOptions::Deterministic, Ok(_vals)) => write!(f, "<pointer>"),
```

Comment:
Do we still want the unused `_vals` here?

Reply:
Changed to just `_`.